From d8f45ebc256e6da36f1c7abaaa62db92e2c957e3 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:23 -0400
Subject: [ethernet] igb: remove return statements for void functions

Message-id: <1409062359-13063-66-git-send-email-sassmann@redhat.com>
Patchwork-id: 89194
O-Subject: [RHEL7.1 PATCH 65/81] igb/ixgbe: remove return statements for void functions
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jeff Kirsher <jeffrey.t.kirsher@intel.com>

Remove useless return statements for void functions which do not need
it.

Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit c43f856cc91c6adcb1b7d521bd03106c26767999)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/e1000_nvm.c b/drivers/net/ethernet/intel/igb/e1000_nvm.c
index 92bcdbe..e8280d0 100644
--- a/drivers/net/ethernet/intel/igb/e1000_nvm.c
+++ b/drivers/net/ethernet/intel/igb/e1000_nvm.c
@@ -798,5 +798,4 @@ etrack_id:
   fw_vers->etrack_id = (eeprom_verh << NVM_ETRACK_SHIFT)
    | eeprom_verl;
  }
- return;
 }
-- 
1.7.1